web_src/js/features/common-button.ts

```diff
@@ -62,7 +62,10 @@ export function initGlobalDeleteButton(): void {
             }
           }
 
-          const response = await POST(btn.getAttribute('data-url'), {data: postData});
+          const method = btn.getAttribute('data-method')?.toUpperCase() || 'POST';
+          const response = method === 'DELETE' ?
+            await DELETE(btn.getAttribute('data-url')) :
+            await POST(btn.getAttribute('data-url'), {data: postData});
```

Comment:
![image](https://github.com/user-attachments/assets/af4aa88a-bd0e-4e98-8f5e-9168c4448ec6)

Don't use this method in new code.

You can use `show-modal` or `link-action (with data-modal-confirm-xxx)`